client/src/components/Navigation/index2.jsx

```diff
@@ -0,0 +1,198 @@
+        dark: "#ebebeb",
+        light: "#444444"
+      }
+    }
+  },
+  button: {
+    border: {
+      radius: "16px"
+    }
+  }
+}
+
+
+
+const Navigation2 = ({ setScreen}) => {
+  const [layer, setLayer] = React.useState()
+  return (
+    
+  <Grommet theme={hivestack}>
+    <Box justify="between" direction="row" background={{"color":"brand"}}>
+      <Box direction="row">
+          <Box align="center" justify="center" pad={{"vertical":"small","horizontal":"medium"}} direction="row" gap="small" fill="vertical">
+          <Heading level='3' margin='none' >
+            <Button label='HiveStack' path='/' plain='true' weight="bold" size="large" color="white">
+            </Button>
+            </Heading>
+            <TextInput size="small" plain={false} type="text"  />
+            <Search color="white"/>
+          </Box>
+        <Button hoverIndicator={true}>
+          <Box align="center" justify="center" pad={{"horizontal":"small","vertical":"medium"}}>
+            <Text color="white">
+              section
+            </Text>
+          </Box>
+        </Button>        <Button hoverIndicator={true}>
+          <Box align="center" justify="center" pad={{"horizontal":"small","vertical":"medium"}}>
+            <Text color="white">
+              section
+            </Text>
+          </Box>
+        </Button>        <Button hoverIndicator={true}>
+          <Box align="center" justify="center" pad={{"horizontal":"small","vertical":"medium"}}>
+            <Text color="white">
+              section
+            </Text>
+          </Box>
+        </Button>        <Button hoverIndicator={true}>
+          <Box align="center" justify="center" pad={{"horizontal":"small","vertical":"medium"}}>
+            <Text color="white">
+              section
+            </Text>
+          </Box>
+        </Button>        <Button hoverIndicator={true}>
+          <Box align="center" justify="center" pad={{"horizontal":"small","vertical":"medium"}}>
+            <Text color="white">
+              section
+            </Text>
+          </Box>
+        </Button>
+      </Box>
+      <Box align="center" justify="center" pad={{"horizontal":"large"}}>
+        <Menu icon={<User />} open={false} items={[{"label":"Profile"},{"label":"Sign Out"}]} />
+      </Box>
+    </Box>
+  </Grommet>
+  )
+}
+
+const screens = {
+  4: Navigation2
+}
+
+
+const NavigationAuth = () => (
+    <Box direction='row' justify='right' gap='small'>
+    <ul>
+        <RoutedButton label="Home" path={ROUTES.HOME} />
+        <RoutedButton label='Sign Out' path={ROUTES.SIGN_OUT_BUTTON} />
+    </ul>
+    </Box>
+  );
+  
+  const NavigationNonAuth = () => (
+    <Box direction='row' justify='right' gap='small'>
+    <ul>
+        <RoutedButton label="Sign In" path={ROUTES.SIGN_IN} />
+        <RoutedButton label="Sign Up" path={ROUTES.SIGN_UP} />
+    </ul>
+    </Box>
+  
+  );
+  
+  export default () => {
+    const [screen, setScreen] = React.useState(4)
```

Comment:
what is the purpose of putting the nav component into a hash table then passing it into this component? If you're trying to conditionally render between the regular nav2 and authNav, there is a much better way to do this then writing separate components. Please clean this up.